plugin/tests/compile-fail/oportunistic-issues.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(plugin)]
+#![plugin(mutagen_plugin)]
+#![feature(custom_attribute)]
+extern crate mutagen;
+
+#[mutate]
+fn main() {
+    let mut count = 0;
+    count += 1; //~  mismatched types [E0308]
+}
```

Comment:
This is really an error. I think we are missing a `&mut` for the left-hand side argument.

Reply:
Yes, adding the `&mut` fixes the issue, at least, on the code I was mutating